package.json

```diff
@@ -24,13 +26,13 @@
         "reset": "pnpm run clean && pnpm run reset:modules",
         "reset:modules": "pnpm -r --parallel --include-workspace-root exec pnpm dlx rimraf node_modules pnpm-lock.yaml",
 
-        "test": "jest --passWithNoTests"
+        "test-all": "pnpm run /^test:.*/",
+        "test:react-router": "cd packages/react-router && pnpm exec jest"
```

Comment:
What do you mean without having it as a dependency?

 `@squide/react-router` have Jest as a dependency.

Reply:
Talked in private about this. While we agree that it's not currently optimal, the alternatives aren't either.

We will revisit this if we choose to add Nx to our stack as it would require so change to integrate the tool.